web-ui/src/main/resources/catalog/components/validationtools/partials/mdValidationTools.html

```diff
@@ -44,7 +44,12 @@
             class="fa fa-chevron-right"/>&nbsp;
 
           <span>{{key}}</span>
-      </a>
+        </a>
+        <!--
```

Comment:
I understand for now requires to update the UI to use the mode attribute? Would be good after merging the pull request to create an issue to improve this using a setting in a future pull request.

Reply:
Yep we have been discussing it with users and we don't really know what should be configurable and how. If using CSW, then record as to be public and if using a portal endpoint, then the record has to be published in that portal filter. So it adds some constraints that it can be hard to figure out by the end user. So upload is easier from that point of view. But yes the config can be added later on.